docs/cookbook/birthdayIn2030.mjs

```diff
@@ -3,5 +3,5 @@ const birthday = Temporal.PlainMonthDay.from('12-15');
 const birthdayIn2030 = birthday.toPlainDate({ year: 2030 });
 birthdayIn2030.dayOfWeek; // => 7
 
-assert(birthdayIn2030 instanceof Temporal.PlainDate);
+assert.ok(birthdayIn2030 instanceof Temporal.PlainDate);
```

Comment:
Are these two not the same?

Reply:
ah, yes, I had an `import * as assert` rather than `import assert` in my codegen. Fixing!